tests/sitemaps_tests/test_generic.py

```diff
@@ -45,3 +45,16 @@ def test_generic_sitemap(self):
 </urlset>
 """ % expected
         self.assertXMLEqual(response.content.decode(), expected_content)
+
+    def test_generic_sitemap_lastmod(self):
+        test_model = TestModel.objects.first()
```

Comment:
`TestModel.objects.update(lastmod=datetime(2013, 3, 13, 10, 0, 0))` might be simpler.